src/hammer-vlsi/hammer_vlsi/constraints.py

```diff
@@ -78,6 +79,20 @@ def from_setting(d: dict) -> "SRAMParameters":
     ('tie', Optional[str])
 ])
 
+BumpAssignment = NamedTuple('BumpAssignment', [
+    ('name', str),
+    ('x', int),
+    ('y', int)
+])
+
+BumpsDefinition = NamedTuple('BumpsDefinition', [
+    ('x', int),
+    ('y', int),
```

Comment:
Is this total `x` and `y` bump counts? Not clear to me how this is different from the `x` and `y` in `BumpAssignment`

Reply:
I think that is the case (`x (int) - number of bumps in the x dimension`).

Maybe we could rename it to `total_x` or something?

Reply:
I actually think `x` and `y` are fine, after thinking about it some more.